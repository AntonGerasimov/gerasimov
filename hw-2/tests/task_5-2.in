12 21 30
12.2 3.3 14.0
