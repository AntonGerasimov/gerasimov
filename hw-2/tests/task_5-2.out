63
29.500000
